backend/bamboo/schemas/site.py

```diff
@@ -0,0 +1,13 @@
+from apiflask import Schema
+from apiflask.fields import Dict, String
+
+
+class SiteIn(Schema):
+    name = String(required=True)
+    config = Dict(required=True)
+
+
+class SiteOut(Schema):
+    id = String()
+    name = String()
+    config = Dict()
```

Comment:
Why only three fields, it should contain all, except for relational fields.

Reply:
Fixed!